monitoring.go

```diff
@@ -260,40 +260,40 @@ func (sc *snowflakeConn) rowsForRunningQuery(
 
 // Wait for query to complete from a query id from /queries/<qid>/result endpoint.
 func (sc *snowflakeConn) blockOnRunningQuery(
-    ctx context.Context, qid string) error {
```

Comment:
Are there functional changes in this block or just white space?

Reply:
This happened because I ran `make lint`